### DISCUSSION TITLE
Not being able to use data store

### DISCUSSION THREAD

**Discussion by gilvanna** [getgauge/gauge#1905] [General]
I am trying to use data store and use some data from one step in another step. 
But, when I try to use 

`import com.thoughtworks.gauge.datastore.ScenarioDataStore;

// Adding value
ScenarioDataStore.put("element-id", "455678");`

as explained in here: hhttps://docs.gauge.org/writing-specifications.html?os=macos&language=java&ide=vscode#data-store

It does not works. 
Looks like this import is not working. 

Is there a different way to use data between steps?

**Comment by zabil** [comment]
The new API's were added in this release
https://github.com/getgauge/gauge-java/releases/tag/v0.7.10

Please make sure that you are using the correct version
